frontend/package.json

```diff
@@ -10,22 +10,20 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "axios": "^0.22.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
-    "typescript": "^4.1.2",
```

Comment:
This is usually not a good practice to upgrade the version of the dependencies our project is actively interacting with. Version-up might change some files in the dependencies and break our project. I am pretty sure you did a test run by yourself and created the pull request after you checked that nothing was broken. Still, next time, please be more careful in upgrading the version by making a separate commit or even a separate branch only for the version change.